types/networks/src/lib.rs

```diff
@@ -19,7 +28,6 @@ pub use self::mainnet::*;
 mod mainnet;
 #[cfg(feature = "conformace")]
 pub use self::mainnet::*;
-pub use crate::mainnet::*;
```

Comment:
I think this might be breaking the test vectors: https://github.com/ChainSafe/forest/runs/4806953786?check_suite_focus=true